augly/image/functional.py

```diff
@@ -1988,6 +1988,7 @@ def resize(
     metadata: Optional[List[Dict[str, Any]]] = None,
     bboxes: Optional[List[Tuple]] = None,
     bbox_format: Optional[str] = None,
+    interpolation: int = Image.BILINEAR
```

Comment:
```suggestion
    interpolation: int = Image.BILINEAR,
    metadata: Optional[List[Dict[str, Any]]] = None,
    bboxes: Optional[List[Tuple]] = None,
    bbox_format: Optional[str] = None,
```

- Augmentation-specific params should go above the standard ones `metadata`, `bboxes`, & `bbox_format`.
- Let's rename `interpolation` to `resample` to match the name of the param we're passing this in as to `PIL.Image.resize()`.
- Also please add a comment below for this new param (on line 2008).